requirements.txt

```
@@ -1,2 +1,2 @@
-urllib3 == 1.24.1
+urllib3 == 1.24.2
```

Al ser sdklib una libreria yo permitiria cualquier version hacia arriba de sus dependencias
```suggestion
urllib3>=1.24.1
```

Ya que esta pegando errores cuando se instala con otro software que necesita esas librerias en versiones superiores:

> ERROR: sdklib 1.10.3 has requirement urllib3==1.22, but you'll have urllib3 1.25.7 which is incompatible.